honeybadgermpc/polynomial.py

```diff
@@ -117,15 +132,12 @@ def fft_helper(A, omega, field):
     list is of the form [a0, a1, ... , an].
     """
     n = len(A)
-
-    # Check if n is a power of 2.
-    assert not (n & (n-1))
+    assert not (n & (n-1)), "n must be a power of 2"
```

Comment:
See comment in https://github.com/initc3/HoneyBadgerMPC/pull/51/files#r216349389

Reply:
I think all of these are actually only for debugging and are correct as asserts (they are not for validating inputs or messages from other parties)

Reply:
Resolving this and other comments on asserts.